lcm-python/meson.build

```
@@ -0,0 +1,28 @@
+py_installation = pymod.find_installation()
+python_site = py_installation.get_path('platlib').replace(py_installation.get_path('data') / '', '')
+python_include = py_installation.get_path('include')
+
+lcm_python_sources = ['module.c', 'pyeventlog.c', 'pylcm.c', 'pylcm_subscription.c']
+
+if windows
+  # lcm_python_lib = py_installation.extension_module('lcm-python', lcm_python_sources,
+  #   dependencies : lcm_static_lib_dep,
+  #   include_directories : include_directories(python_include),
+  #   install : true)
+  message('Python not currently supported in this LCM build for Windows')
+elif host_machine.system() == 'darwin' #macos
+  lcm_python_lib = py_installation.extension_module('lcm-python', lcm_python_sources,
+    dependencies : lcm_static_lib_dep,
+    include_directories : include_directories(python_include),
+    link_args : '-undefined dynamic_lookup -Wl,-no_fixup_chains',
+    install : true)
+else
+  lcm_python_lib = py_installation.extension_module('lcm-python', lcm_python_sources,
```

It might make more sense to just call the Python module `lcm`

```suggestion
  # lcm_python_lib = py_installation.extension_module('lcm', lcm_python_sources,
  #   dependencies : lcm_static_lib_dep,
  #   include_directories : include_directories(python_include),
  #   install : true)
  message('Python not currently supported in this LCM build for Windows')
elif host_machine.system() == 'darwin' #macos
  lcm_python_lib = py_installation.extension_module('lcm', lcm_python_sources,
    dependencies : lcm_static_lib_dep,
    include_directories : include_directories(python_include),
    link_args : '-undefined dynamic_lookup -Wl,-no_fixup_chains',
    install : true)
else
  lcm_python_lib = py_installation.extension_module('lcm', lcm_python_sources,
```

---

Didn't realize that was possible, but it's definitely better. Renamed: 5d8ff8e471920589e165446f7d85078b3f39a84e